jerry-core/ecma/operations/ecma-exceptions.c

```diff
@@ -295,6 +295,8 @@ ecma_get_error_type (ecma_object_t *error_object_p) /**< possible error object *
 /**
  * Raise a standard ecma-error with the given type and message.
  *
+ * Note: The given buffer must be CESU8 encoded and the buffer must exist until the error message is not freed.
+ *
```

Comment:
The `char *` message argument should be replaced with `ecma_error_msg_t`, and this function should call `ecma_get_error_msg`. This guarantees that the string is a magic string and it exists statically, and then there's no reason for the note. Also the string size can then be retrieved, without the need for calling `lit_zt_utf8_string_size`.